packages/tradershub/src/features/cfd/modals/TradeModal/TradeModal.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FC, useEffect } from 'react';
+import QRCode from 'qrcode.react';
+import { Provider } from '@deriv/library';
+import { Text, useBreakpoint } from '@deriv/quill-design';
+import { Modal } from '../../../../components/Modal';
+import InstallationAppleIcon from '../../../../public/images/ic-installation-apple.svg';
+import InstallationGoogleIcon from '../../../../public/images/ic-installation-google.svg';
+import InstallationHuaweiIcon from '../../../../public/images/ic-installation-huawei.svg';
+import { THooks, TMarketTypes, TPlatforms } from '../../../../types';
+import { PlatformDetails } from '../../constants';
+import { MT5TradeScreen } from '../../screens/MT5TradeScreen';
+
+type TAppLinks = {
+    android: string;
+    huawei?: string;
+    ios: string;
+};
+
+const LinksMapper: Record<TPlatforms.All, TAppLinks> = {
```

Comment:
@sergei-deriv please move this to the `packages/tradershub/src/features/cfd/constants.tsx` path